backend/src/integration-test/kotlin/com/tngtech/apicenter/backend/connector/rest/controller/VersionControllerIntegrationTest.kt

```diff
@@ -25,15 +25,24 @@ class VersionControllerIntegrationTest {
 
     @Test
     fun findOneVersion_shouldReturnVersion() {
-        mockMvc.perform(get("/specifications/b6b06513-d259-4faf-b34b-a216b3daad6a/versions/v1")
+        mockMvc.perform(get("/api/1.0/specifications/b6b06513-d259-4faf-b34b-a216b3daad6a/v1")
```

Comment:
I think I'd rather keep the `/versions/` bit, so that you can get a list of all versions from there.

Reply:
Sorry I don't understand. How does `/versions/` add meaning? The resource path is uniquely identifiable by the specID and version alone. I don't mind changing it, but would like to know why

Reply:
While I agree that it looks unnecessary, I think I only ever saw REST interfaces with the "parameter-names" added. For example, if I make a GET call to `/specifications/{id}`, I get that resource. If I would make one to `/specifications/{id}/versions` I'd expect to get a list of all the versions. If we don't have the `versions` part anymore, where do I get this list?

Reply:
`/specifications/{specificationId}/versions` was never an API endpoint. (I had not made any edits to api-definition.json until now.) 😕

Reply:
Doesn't have to be right now, but at least we have the opportunity.